cli/tests/e2e/snapshots/test_ci/test_github_ci_bad_base_sha/results.txt

```diff
@@ -15,15 +15,15 @@ Findings:
         useless comparison to 5
 
          ▶▶┆ Autofix ▶ bar == 2
-          1┆ bar == 2
+          1┆ bar == 5
```

Comment:
Autofix used to be "on" for this test because it was mocked and thus its value was truthy, even though by default it should have been False. The changes in this test and others like it are thus expected.